### DISCUSSION TITLE
Remove steering behavior of a crowd agent

### DISCUSSION THREAD

**Discussion by Samsy** [isaac-mason/recast-navigation-js#271] [General]
Hey hello!  

Thanks a lot for this package, looks wonderful.

I've got a crowd agent working, is there a way to remove the steering / 'sliding' behavior of the agents ?

**Comment by isaac-mason** [comment]
Hi @Samsy!

Using `Crowd` you do have to surrender a fair bit of control over how agents move to the crowd manager. You can see the CrowdAgentParams type docs to see available configuration options: https://docs.recast-navigation-js.isaacmason.com/types/index.CrowdAgentParams.html

What in particular are you referring to by "steering" / "sliding"? If you can share a video or something to help describe that could be handy to understand too.

**Comment by Samsy** [comment]
Hello @isaac-mason 

Thanks for your answer, 

This is what I meant with steering / sliding.

Whenever an agent gets to a point or change position, it is kind of lerping / rebounding around the target position. But not only when it reaches it, but when it goes on the way to the target position the agent is sliding

https://github.com/isaac-mason/recast-navigation-js/assets/3786958/23e18353-65ce-45c6-8c91-b3836abb02af

**Reply by isaac-mason** [reply]
Ah thanks for the clarification @Samsy. This is probably due to how I've configured max acceleration and max speed in the examples.

**Reply by isaac-mason** [reply]
If you want to avoid the "rebounding" behaviour where the agent reaches the target but then moves past it, you can set a higher agent max acceleration. My understanding is that max acceleration also controls deceleration.

**Reply by Samsy** [reply]
seems to work ! 

Thank you very much !
